server/memory_store.go

```diff
@@ -1,4 +1,4 @@
-package store
+package server
 
 import (
```

Comment:
What's the rationale for overloading the server package with storage interface implementations. IMHO it's more clean to keep to get this abstractions separated by package